diracx/values.yaml

```diff
@@ -183,6 +183,86 @@ rabbitmq:
 
 ##########################
 
+opentelemetry-collector:
+  enabled: true
+  mode: daemonset
+  #internalTrafficPolicy:Local
+  presets:
+    # enables the k8sattributesprocessor and adds it to the traces, metrics, and logs pipelines
+    kubernetesAttributes:
+      enabled: false
+    # enables the kubeletstatsreceiver and adds it to the metrics pipelines
+    kubeletMetrics:
+      enabled: false
+    # Enables the filelogreceiver and adds it to the logs pipelines
+    logsCollection:
+      enabled: false
+
+  #CHRIS example taken from https://opentelemetry.io/docs/collector/configuration/
+  config:
+    exporters:
+      otlp/jaeger:
+        endpoint: diracx-demo-jaeger-collector:4317
+        tls:
+          insecure: true
+      logging:
+        loglevel: debug
+    service:
+      pipelines:
+        traces:
+          exporters: [ otlp/jaeger, logging ]
+        metrics:
+          exporters: [ logging ]
+        logs:
+          exporters: [ logging ]
+
+
+##########################
+
+
+jaeger:
+  enabled: true
+  # service:
+  #   type: NodePort
+  #   port: 16686
+  #   nodePort: 32003
+
+  provisionDataStore:
+    cassandra: false
+  allInOne:
+    enabled: true
+  storage:
+    type: none
+  agent:
+    enabled: false
+  collector:
+    enabled: false
+  query:
+    enabled: false
+
+
+##########################
+grafana:
+  enabled: true
+  # To get the password for admin
+  # kubectl get secrets diracx-demo-grafana -o jsonpath="{.data.admin-password}" | base64 --decode ; echo
```

Comment:
We have this with RabbitMQ as well, I think we should embed them in the demo `values.yaml` as it makes helm upgrade hard to use.

Reply:
embed what ?

Reply:
The secrets